pipeline/create-tsa-options.ps1

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can we move this file to under the tools folder? We'll also need to update the path in the yaml file

Reply:
moved to tools folder